pallets/rmrk-core/src/benchmarking.rs

```diff
@@ -0,0 +1,443 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+// Benchmarks for rmrk-core pallet
+
+use super::*;
+#[allow(unused)]
+use crate::Pallet as RmrkCore;
+
+use frame_benchmarking::{account, benchmarks, whitelisted_caller};
+use frame_support::traits::{Currency, Get};
+use frame_system::RawOrigin;
+use rmrk_traits::{AccountIdOrCollectionNftTuple, BasicResource};
+use sp_runtime::traits::Bounded;
+use sp_std::vec;
+
+pub type BalanceOf<T> = <<T as pallet_uniques::Config>::Currency as Currency<
+	<T as frame_system::Config>::AccountId,
+>>::Balance;
+
+const SEED: u32 = 0;
+
+macro_rules! bvec {
+	($( $x:tt )*) => {
+		vec![$( $x )*].try_into().unwrap()
+	}
+}
+
+/// Turns a string into a BoundedVec
+fn stb<T: Config>(s: &str) -> BoundedVec<u8, T::ValueLimit> {
+	s.as_bytes().to_vec().try_into().unwrap()
+}
+
+/// Turns a string into a BoundedVec
+fn stbk<T: Config>(s: &str) -> BoundedVec<u8, T::KeyLimit> {
+	s.as_bytes().to_vec().try_into().unwrap()
+}
+
+/// Turns a string into a BoundedVec
+fn stbd<T: Config>(s: &str) -> StringLimitOf<T> {
+	s.as_bytes().to_vec().try_into().unwrap()
+}
+
+fn funded_account<T: Config>(name: &'static str, index: u32) -> T::AccountId {
+	let caller: T::AccountId = account(name, index, SEED);
+	T::Currency::make_free_balance_be(&caller, BalanceOf::<T>::max_value());
+	caller
+}
+
+/// Assert that the last event equals the provided one.
+fn assert_last_event<T: Config>(generic_event: <T as Config>::Event) {
+	frame_system::Pallet::<T>::assert_last_event(generic_event.into());
+}
+
+/// Assert that the event was fired.
+// fn assert_has_event<T: Config>(generic_event: <T as Config>::Event) {
```

Comment:
Is this a placeholder for future?

Reply:
yes, but since it is not used I'll remove it